config/rabbitmq.php

```diff
@@ -39,6 +39,7 @@
         'durable'     => env('RABBITMQ_QUEUE_DURABLE', true),
         'exclusive'   => env('RABBITMQ_QUEUE_EXCLUSIVE', false),
         'auto_delete' => env('RABBITMQ_QUEUE_AUTODELETE', false),
+        'arguments'   => json_decode(env('RABBITMQ_QUEUE_ARGUMENTS', '{}'), 1),
```

Comment:
Updated. I added a new variable `$configQueueArguments`. I don't know how you feel about adding it, since I could just: 

`$this->configQueue['arguments'] = json_decode($this->configQueue['arguments'], 1)`

..  it didn't seem ok because it overwrites the original value. So I opted to create a new variable. If you want it differently just say, I'm ok with either (or something else that you prefer).

Did you see my comment in tests?